Fundsoft/report.md

```diff
@@ -131,8 +131,25 @@ What are your main take-aways from this project? What did you learn?
+
+
+Refactoring plan for do_update(): 
+
+More about ESSENCE alphas:
+* Oppurtunity
+    * The issue that we solved was requested by several people already utilizing the pdm open source library. Thus there existed a "customer" need for the patch that we created. The articulated needs of the users were the foundation for our patch development and justified the reason for implementing it. We avoided implementing something that was not requested which can sometimes happen with large software companies/games where the company implements features that no one requested and actively ignore customer input which usually results in a worse product or customer experience. 
+* Stakeholders
+    * Since pdm is an open source project there is a large overlap between the stakeholders of the project and the project customers. Many of the people that use the library also contribute to its continuous evolution and improvement. Thus improving the library ultimately improves ones own work which creates a great incentive to contribute as well as ensure that patches are of high quality and throughly tested. 
+* Requirements
+    * Again since pdm is an open source project with many different contributors it can sometimes be difficult to understand the specific requirements for a given issue. Thankfully our issue was well explained with a concensus on how it should work and be implemented. 
```

Comment:
This is for the most part true. However, we did have to come up with a few requirements of our own which we did by analyzing the code.